Tests/Editor/Unit/Extension Methods/TransformExtensionsTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NUnit.Framework;
+using UnityEngine;
+using UnityEditor.Experimental.EditorVR.Extensions;
+
+namespace UnityEditor.Experimental.EditorVR.Tests.Extensions
+{
+    [TestFixture]
+    public class TransformExtensionsTests
+    {
+        GameObject go;
+
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            go = new GameObject();
+        }
+
+        [Test]
+        public void TransformBounds_TranslatesLocalBoundsToWorld()
+        {
+            var size = new Vector3(2, 2, 2);
+            var offset = new Vector3(1, 2, 3);
+            go.transform.position += offset;
+            var localBounds = new Bounds(go.transform.position, size);
+            var worldBounds = go.transform.TransformBounds(localBounds);
+
+            Assert.AreEqual(localBounds.center, worldBounds.center - offset);
```

Comment:
For these asserts specifically we might want to test to a really small delta instead of an exact equality.  We are transforming vectors through bounds and while it is working now it is just a little more future proof.